pcp/screens/filesys

```diff
@@ -0,0 +1,77 @@
+#
+# pcp-htop(1) configuration file - see pcp-htop(5)
+#
+
+[filesys]
+caption = Filesystem
+enabled = true
```

Comment:
For me, this screen doesn't function correctly.  The headings don't seem to match the data displayed.